.travis.yml

```diff
@@ -2,13 +2,20 @@ language: cpp
 
 dist: trusty
 
-apt:
-  addons:
-    sources:
-      - george-edison55-cmake-3.x
-    packages:
-      - cmake
-      - cmake-data
+matrix:
+    include:
+        - os: linux
+          addons:
+            apt:
+                sources:
+                    - ubuntu-toolchain-r-test
+                    - george-edison55-cmake-3.x
+                packages:
+                    - cmake
+                    - cmake-data
+                    - g++-5
+          env:
+             - MATRIX_EVAL="CC=gcc-5 && CXX=g++-5"
```

Comment:
Hi Arif, please do not include this change. We need to support GCC 4.8 and above (ROOT depends on VecCore and needs to support GCC 4.8 as well).

Reply:
I will make an update to the build matrix to contain more compiler versions and push it, so that we can test.

Reply:
No problem, you couldn't know it. In any case, I will expand the build matrix this week.
I am working on it on my fork and will push when I'm done.